source/includes/_tasks.md

```diff
@@ -0,0 +1,55 @@
+# Tasks
+
+## Get all tasks for current user
+
+```http
+GET /tasks HTTP/1.1
+Host: api.company-mood.com
+Content-Type: application/json
+Accept: application/vnd.company-mood-v2+json
+Authorization: Bearer 795665b4-53da-468c-a0d7-ab2d82e58406
+X-App-Token: 27f50875-9a43-4d6c-a376-6968f09858db
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "data": [
+     {
+       "id": "f7559f2a-8f1c-461e-8a9b-7efea5564edb",
+       "type": "tasks",
+       "attributes": {
+         "name": "Implement CompanyMood",
+         "description": "Implement continious Mood monitoring",
+         "progress": '100%',
```

Comment:
There is no DB attribute named `progress`
We have the field `percent` which is an integer.